app/templates/search.hbs

```diff
@@ -1,2 +1,40 @@
-Search Page
-{{outlet}}
+
+<div id="bg">
+  <img src="https://s3-us-west-1.amazonaws.com/waldojobs-dev/app-assets/images/source/welcomeBG3.jpg"/>
+</div>
+<div class="wrapper">
+<img class="search-page-logo" src="https://s3-us-west-1.amazonaws.com/waldojobs-dev/app-assets/images/source/waldo_logoColor.png"/>
+
+<h3 class="search-page-slogan"> Let's find the job you've been looking for. </h3>
+
+<div class="div-search-bar">
+{{search-bar
+  placeholder="Search For Jobs"
+  image="https://png.icons8.com/search-filled/ios7/39/00B5AF"
+}}
+</div>
+<div id="recommended-jobs-header">
+   <h3 id="recommended-jobs-title"> Our Top Picks for You. </h3>
+    <div {{action "showRecommendedJobSlider"}}><img src="https://png.icons8.com/back-to/color/50/e74c3c"></div>
+    <div id="recommended-jobs-body" class="show">
+        {{#each model as |user|}}
+
+        <div class="list">
```

Comment:
I believe Jeremy is making a component for a job search listing, so at some point, we can come back and make this use his component 👍